### PR TITLE
Add Parameter description in 2016 version

### PR DESCRIPTION
**Pull request by velkovb**
Added description for all parameters in 2016 version and fixed few missing in 2012 R2 version.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/failoverclusters/remove-clustervmmonitoreditem.md [hunk 1]
```diff
@@ -122,7 +123,7 @@ Accept wildcard characters: False
 ```
 
 ### -InputObject
-
+Specifies the cluster on which to run the cmdlet, the clustered virtual machine from which to remove monitoring, or the clustered virtual machine monitored item object to stop monitoring
```
Comment (JohanFreelancer9) [inline docset/windows/failoverclusters/remove-clustervmmonitoreditem.md]:
```suggestion
Specifies the cluster on which to run the cmdlet, the clustered virtual machine from which to remove monitoring, or the clustered virtual machine monitored item to stop monitoring.
```

File: docset/winserver2012r2-ps/failoverclusters/Remove-ClusterVMMonitoredItem.md [hunk 2]
```diff
@@ -198,7 +200,7 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/failoverclusters/Remove-ClusterVMMonitoredItem.md]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [About CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```

Comment (velkovb) [reply docset/winserver2012r2-ps/failoverclusters/Remove-ClusterVMMonitoredItem.md]: Actual tiltle is about_CommonParameters:
image
Leaving it as is.

**Pull request merged**